module/PowerNSX.psm1

```diff
@@ -22973,7 +22973,13 @@ function Add-NsxSecurityGroupMember {
 
     A valid PowerCLI session is required to pass certain types of objects
     supported by the IncludeMember and ExcludeMember parameters.
+    
+    .EXAMPLE
+    $vms = @(Get-VM -Name vmname*) # fetch the VMs to be added to the NSX security group
+    $sg = Get-NsxSecurityGroup -name "securitygroupname" # fetch the Securty group
+    Add-NsxSecurityGroupMember -SecurityGroup $sg -Member $vms -Verbose
 
+    This example shows how to add a bunch of vms to a NSX Security group.
```

Comment:
Make a better example for exchange Add All VM named "XXX- on security SG-XXX ?

add some other example with other SecurityGroup possibilité ?

Reply:
@alagoutte  working on them now , was out of  git for some time

Reply:
sure , i will